jc/parsers/curl_head.py

```diff
@@ -0,0 +1,123 @@
+"""jc - JSON Convert `Curl` string parser
+
+Supports output of Curl when run with the HEAD option (-I)
+
+Usage (cli):
+
+    $ curl --head example.org | jc --curl-head
+
+Usage (module):
+
+    import jc
+    result = jc.parse('curl-head', curl_output)
+
+Schema:
+
+Curl output converted to a dictionary
+
+    {
+      "key1":       string,
+      "key2":       string
+    }
+
+Examples:
+
+curl -s -I google.com | python3 -m jc --curl-head | jq
```

Comment:
We'll want to clean up the example command.

Reply:
fixed 987ff97